pkg/http/api/v1/handler.go

```diff
@@ -187,6 +191,8 @@ func (h *requestHandler) GetScanReport(res http.ResponseWriter, req *http.Reques
 }
 
 func (h *requestHandler) GetMetadata(res http.ResponseWriter, req *http.Request) {
+	vi, _ := h.wrapper.GetVersion()
```

Comment:
Handle this error. See the default error handler helpers in BaseHandler

Reply:
Fixed https://github.com/aquasecurity/harbor-scanner-trivy/pull/87/commits/7ab2020862901d69d91bacee72f88a78abd8feee